app/assets/javascripts/replies.js

```diff
@@ -0,0 +1,30 @@
+jQuery(function() {
+
+  var current_selection = "";
+
+  jQuery(document).on('mouseup', 'body', function() {
+    current_selection = window.getSelection().toString();
+  })
+
+  jQuery('.split-off-ticket').bind('click', function(event) {
+    if (current_selection != "") {
+      var url = $(this).attr('href') + ".json";
+      var data = {
+        selected_text: current_selection
+      }
+
+      jQuery.ajax({
+        url: url,
+        type: 'post',
+        data: data,
+        success: function(result) {
+          window.location = result.ticket_path;
+        }
+      });
+
+      event.stopPropagation();
+      event.preventDefault();
+      return false;
```

Comment:
Thanks for the review. Indeed, three lines seem like overkill. For some reason, using just `event.preventDefault()` did not work for me in Safari. But using just `return false` does the trick. -> https://github.com/ivaldi/brimir/pull/373/commits/cea9b8735cd25fe543e3eeb0fbdce3ff5299475d